src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -645,6 +645,12 @@ public static boolean changeOrderStatus(PurchaseOrder purchaseOrder, List<PoLine
     return isUpdateRequired;
   }
 
+  public static String convertTagListToCqlQuery(Collection<String> values, String fieldName, boolean strictMatch) {
+
+    String prefix = fieldName + (strictMatch ? "==(\"" : "=(\"");
+    return StreamEx.of(values).joining("\" or \"", prefix, "\")");
+  }
+
```

Comment:
Does this new logic testes in some way using integration or karate tests?
For example, test can create order with assigning tags, and after this checking if tags created